<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HardwareWidget</class>
 <widget class="QWidget" name="HardwareWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>653</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="cvInGroup">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Knobs / CV in</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="1" column="2">
       <widget class="QLabel" name="cvInLabel3">
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QDial" name="cvInKnob1">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="cvInLabel1">
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDial" name="cvInKnob4">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDial" name="cvInKnob2">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="cvInLabel2">
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="cvInLabel4">
        <property name="text">
         <string>000</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QDial" name="cvInKnob3">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QGroupBox" name="gateInGroup">
     <property name="title">
      <string>Gates</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <widget class="QRadioButton" name="gateInCheck1">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>33</y>
        <width>19</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="gateInCheck2">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>57</y>
        <width>19</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>33</y>
        <width>23</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>one</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>42</x>
        <y>57</y>
        <width>23</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>two</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="DaisyLabel" name="daisyInfo">
     <property name="minimumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>80</width>
       <height>80</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QGroupBox" name="stateGroup">
     <property name="title">
      <string>Audio State</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Sample Rate</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="sampleRateLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>???</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Block Size</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="blockSizeLabel">
        <property name="text">
         <string>???</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="encoderGroup">
     <property name="title">
      <string>Encoder</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="encoderBackButton">
        <property name="text">
         <string>&lt;</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="encoderForwardButton">
        <property name="text">
         <string>&gt;</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="encoderClickButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>* click * </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QPushButton" name="encoderPressButton">
        <property name="text">
         <string> _press_</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLabel" name="displayLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>192</width>
       <height>96</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>192</width>
       <height>96</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QGroupBox" name="groupBox_6">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Gate / LED</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QRadioButton" name="gateOutCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>gate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ledCheck">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoExclusive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>LED</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QGroupBox" name="cvOutGroup">
     <property name="title">
      <string>CV out</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QSlider" name="cvOutSlider1">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="cvOutLabel1">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="cvOutLabel2">
        <property name="text">
         <string>0.00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QSlider" name="cvOutSlider2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
        <property name="tracking">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="audioInGroup">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Audio In</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="7" column="0" colspan="3">
       <widget class="AudioBufferLabel" name="waveInLabel4">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="AudioBufferLabel" name="waveInLabel2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_14">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>four</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="AudioBufferLabel" name="waveInLabel3">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="AudioBufferLabel" name="waveInLabel1">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_12">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>three</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLabel" name="oscilatorLabel4">
        <property name="text">
         <string>???</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="oscilatorLabel3">
        <property name="text">
         <string>???</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLabel" name="oscilatorLabel2">
        <property name="text">
         <string>???</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLabel" name="oscilatorLabel1">
        <property name="text">
         <string>???</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QGroupBox" name="audioOutGroup">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Audio Out</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_15">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>one</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AudioBufferLabel" name="waveOutLabel1">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_17">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>two</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AudioBufferLabel" name="waveOutLabel2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_19">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>three</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AudioBufferLabel" name="waveOutLabel3">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_21">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>four</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AudioBufferLabel" name="waveOutLabel4">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>50</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AudioBufferLabel</class>
   <extends>QLabel</extends>
   <header location="global">AudioBufferLabel.h</header>
  </customwidget>
  <customwidget>
   <class>DaisyLabel</class>
   <extends>QLabel</extends>
   <header location="global">DaisyLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
